src/Controller/BaseTalkController.php

```diff
@@ -31,14 +31,16 @@ class BaseTalkController extends BaseApiController
      * @var EventMapper
      */
     protected $event_mapper;
+
     /**
      * @var TalkMapper
      */
-    private $talk_mapper;
+    public $talk_mapper;
```

Comment:
What do you think about adding tests to prevent these kinds of regressions, perhaps something along the lines of:

```diff
diff --git a/tests/Controller/TalkLinkControllerTest.php b/tests/Controller/TalkLinkControllerTest.php
index 710aeb1..da9fbb3 100644
--- a/tests/Controller/TalkLinkControllerTest.php
+++ b/tests/Controller/TalkLinkControllerTest.php
@@ -10,6 +10,14 @@ use Teapot\StatusCode\Http;

 class TalkLinkControllerTest extends TalkBase
 {
+    public function testDefaults(): void
+    {
+        $controller = new TalkLinkController();
+
+        self::assertNull($controller->talk_mapper);
+        self::assertNull($controller->user_mapper);
+    }
+
     /**
      * Test sending delete link where the link id is not found
      */
diff --git a/tests/Model/ClientModelTest.php b/tests/Model/ClientModelTest.php
new file mode 100644
index 0000000..af51bad
--- /dev/null
+++ b/tests/Model/ClientModelTest.php
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Test\Model;
+
+use Joindin\Api\Model\ClientModel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Joindin\Api\Model\ClientModel
+ */
+final class ClientModelTest extends TestCase
+{
+    public function testDefaults(): void
+    {
+        $model = new ClientModel([]);
+
+        self::assertNull($model->id);
+    }
+}
diff --git a/tests/Model/EventCommentReportModelTest.php b/tests/Model/EventCommentReportModelTest.php
new file mode 100644
index 0000000..c1bbd3d
--- /dev/null
+++ b/tests/Model/EventCommentReportModelTest.php
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Test\Model;
+
+use Joindin\Api\Model\EventCommentReportModel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Joindin\Api\Model\EventCommentReportModel
+ */
+final class EventCommentReportModelTest extends TestCase
+{
+    public function testDefaults(): void
+    {
+        $model = new EventCommentReportModel([]);
+
+        self::assertNull($model->event_id);
+        self::assertNull($model->reporting_user_id);
+    }
+}
diff --git a/tests/Model/TalkCommentReportModelTest.php b/tests/Model/TalkCommentReportModelTest.php
new file mode 100644
index 0000000..4702c08
--- /dev/null
+++ b/tests/Model/TalkCommentReportModelTest.php
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Test\Model;
+
+use Joindin\Api\Model\TalkCommentReportModel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Joindin\Api\Model\EventCommentReportModel
+ */
+final class TalkCommentReportModelTest extends TestCase
+{
+    public function testDefaults(): void
+    {
+        $model = new TalkCommentReportModel([]);
+
+        self::assertNull($model->event_id);
+        self::assertNull($model->reporting_user_id);
+        self::assertNull($model->talk_id);
+    }
+}
diff --git a/tests/Model/TalkModelTest.php b/tests/Model/TalkModelTest.php
new file mode 100644
index 0000000..883ae93
--- /dev/null
+++ b/tests/Model/TalkModelTest.php
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Test\Model;
+
+use Joindin\Api\Model\TalkModel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Joindin\Api\Model\TalkModel
+ */
+final class TalkModelTest extends TestCase
+{
+    public function testDefaults(): void
+    {
+        $model = new TalkModel([]);
+
+        self::assertNull($model->event_id);
+        self::assertNull($model->ID);
+        self::assertNull($model->stub);
+    }
+}
diff --git a/tests/Model/TokenModelTest.php b/tests/Model/TokenModelTest.php
new file mode 100644
index 0000000..c1a5aa6
--- /dev/null
+++ b/tests/Model/TokenModelTest.php
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Test\Model;
+
+use Joindin\Api\Model\TokenModel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Joindin\Api\Model\TokenModel
+ */
+final class TokenModelTest extends TestCase
+{
+    public function testDefaults(): void
+    {
+        $model = new TokenModel([]);
+
+        self::assertNull($model->id);
+    }
+}
diff --git a/tests/Model/TwitterRequestTokenModelTest.php b/tests/Model/TwitterRequestTokenModelTest.php
new file mode 100644
index 0000000..001b86b
--- /dev/null
+++ b/tests/Model/TwitterRequestTokenModelTest.php
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Test\Model;
+
+use Joindin\Api\Model\TwitterRequestTokenModel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Joindin\Api\Model\TwitterRequestTokenModel
+ */
+final class TwitterRequestTokenModelTest extends TestCase
+{
+    public function testDefaults(): void
+    {
+        $model = new TwitterRequestTokenModel([]);
+
+        self::assertNull($model->ID);
+    }
+}
```